src/style.css

```diff
@@ -1,3 +1,67 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  justify-content: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  text-transform: uppercase;
+  margin-left: 20px;
```

Comment:
Be consistent with your vertical margins. It's recommended to add only top or only bottom margin, but not both. In your case, `.nav__item` has a left margin, which is horizontal, but keep the principle in mind for vertical margins.